imagebuildah/util.go

```diff
@@ -111,3 +111,23 @@ func TempDirForURL(dir, prefix, url string) (name string, subdir string, err err
 func InitReexec() bool {
 	return buildah.InitReexec()
 }
+
+// ReposToMap parses the specified repotags and returns a map with repositories
+// as keys and the corresponding arrays of tags as values.
+func ReposToMap(repotags []string) map[string][]string {
+	// map format is repo -> tag
+	repos := make(map[string][]string)
+	for _, repo := range repotags {
+		var repository, tag string
+		if len(repo) > 0 {
+			li := strings.LastIndex(repo, ":")
```

Comment:
What happens if repo does not have a ":"?

Reply:
use `<none>` as a tag? in what situation the `repo` without ":" will be fed to this function, the default tag it has is `:latest`?
```go
        var repository, tag string
        if strings.Contains(repo, ":") {
            li := strings.LastIndex(repo, ":")
            repository = repo[0:li]
            tag = repo[li+1:]
        }
        if !strings.Contains(repo, ":") && len(repo) > 0 {
            repository = repo
            tag = "<none>"
        }
        repos[repository] = append(repos[repository], tag)
```

Reply:
```
        var repository, tag string
        if strings.Contains(repo, ":") {
            li := strings.LastIndex(repo, ":")
            repository = repo[0:li]
            tag = repo[li+1:]
        } else if len(repo) > 0 {
            repository = repo
            tag = "<none>"
        }
        repos[repository] = append(repos[repository], tag)
```

But should we return an error if len(repo) == ""?